data/shared/citizen/scripting/v8/timer.js

```diff
@@ -85,10 +85,18 @@
         animationFrames[animationFrames.length] = callback;
     }
 
-    function setInterval(callback, interval) {
+    function setInterval(callback, interval, ...argsForCallback) {
         const id = nextTimerId();
 
-        setTimer(id, callback, interval);
+        setTimer(
+            id,
+            function() {
+                try {
+                    callback(...argsForCallback);
+                } catch { }
```

Comment:
This is bad. If something will throw an error inside `callback` we won't ever notice it failing in console. I'd better write `setTimer(id, callback.bind(null, ...argsForCallback), interval);`

Reply:
try and catch is there for that. Just add a print in catch.

Reply:
And you will do what already is there. See https://github.com/citizenfx/fivem/blob/master/data/shared/citizen/scripting/v8/timer.js#L127-L131

Reply:
Thing that process every timer callback is already doing that, why doing it here again?

Reply:
I did not pay attention, thx :)